pkg/input/tf.go

```diff
@@ -81,7 +81,11 @@ func newHclConfiguration(moduleTree *hcl_interpreter.ModuleTree) (*HclConfigurat
 		return nil, fmt.Errorf("%w: %v", FailedToParseInput, err)
 	}
 
-	resources := evaluation.Resources()
+	resources := []models.ResourceState{}
+	for _, resource := range evaluation.Resources() {
+		resources = append(resources, resource.Model)
+	}
+
```

Comment:
Nit - it might be worth initializing this slice with `make` since we know its size:
```
	resources := evaluation.Resources()
	resourceStates := make([]models.ResourceState, len(resources))
```